subsys/net/lib/mqtt_sn/Kconfig

```diff
@@ -0,0 +1,52 @@
+# Socket MQTT Library for Zephyr
+
+# Copyright (c) 2018 Nordic Semiconductor ASA
```

Comment:
Please update copyright (here and some other files)

Reply:
Done.